operator/api/v1alpha1/central_types.go

```diff
@@ -422,6 +422,14 @@ type DBPersistentVolumeClaim struct {
 	StorageClassName *string `json:"storageClassName,omitempty"`
 }
 
+// GetClaimName returns the ClaimName for PVC
+func (pvc *DBPersistentVolumeClaim) GetClaimName() *string {
```

Comment:
Seems unused. Drop?

Reply:
Yep, this one is a left over, will delete.